native/c/zusftype.h

```diff
@@ -45,6 +45,7 @@ typedef struct
 
   ZDIAG diag;
 
+  bool created;
```

Comment:
These structures must be double-word aligned (4 byte boundary):

```suggestion
  bool created;
  char _pad[3];
```